### PR TITLE
[CFM-1149] Theme "group" view modes

### PR DESCRIPTION
**Pull request by nirgn975**
https://issuetracker.amplexor.com/jira/browse/CFM-1149

### CODE REVIEW

**Comment by nirgn975** [comment]
capacity4more

**Comment by nirgn975** [comment]
capacity4more

**Comment by nirgn975** [comment]
capacity4more

**Comment by nirgn975** [comment]
@ordavidil Ready for review.

File: project/profiles/capacity4more/modules/c4m/content/c4m_content_group/c4m_content_group.module [hunk 1]
```diff
@@ -842,4 +855,23 @@ function c4m_content_group_node_view($node, $view_mode, $langcode) {
...
+    }
+
+    $variables = array(
+      'root' => $topic,
+      'tree' => implode(', ', $topic_names),
+    );
+    $node->content['c4m_vocab_topic'][$index]['#markup'] = theme('c4m_og_vocab', $variables);
+
+  }
+}
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/content/c4m_content_group/c4m_content_group.module]: @nirgn975 
new line is missing.

**Comment by nirgn975** [comment]
@ordavidil Ready for review.

File: project/profiles/capacity4more/modules/c4m/content/c4m_content_group/c4m_content_group.module [hunk 2]
```diff
@@ -842,4 +855,23 @@ function c4m_content_group_node_view($node, $view_mode, $langcode) {
...
   );
+
+  // Style the Topics and Regions vocab.
+  foreach($node->content['c4m_vocab_topic']['#items'] as $index => $term) {
+    $topic = $term['taxonomy_term'];
+
+    // Create the topics tree.
+    $topic_names = array();
+    $tree = hierarchical_taxonomy_get_tree($topic->vid, $topic->tid);
+    foreach ($tree as $item => $data) {
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/content/c4m_content_group/c4m_content_group.module]: `$data` => `$term`

File: project/profiles/capacity4more/modules/c4m/content/c4m_content_group/c4m_content_group.module [hunk 3]
```diff
@@ -842,4 +855,23 @@ function c4m_content_group_node_view($node, $view_mode, $langcode) {
     '#title' => t('Group administrators'),
     '#markup' => implode(",", $links),
   );
+
+  // Style the Topics and Regions vocab.
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/content/c4m_content_group/c4m_content_group.module]: @nirgn975 
It seems you relate only to topics and not to regions at all

File: project/profiles/capacity4more/modules/c4m/content/c4m_content_group/templates/c4m_og_vocab.tpl.php [hunk 4]
```diff
@@ -0,0 +1,14 @@
+<?php
+/**
+ * @file
+ * Template for the Vocabulary topics and regions.
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/content/c4m_content_group/templates/c4m_og_vocab.tpl.php]: @nirgn975 
It seems you relate only to topics and not to regions at all

File: project/profiles/capacity4more/modules/c4m/content/c4m_content_group/c4m_content_group.module [hunk 5]
```diff
@@ -842,4 +855,23 @@ function c4m_content_group_node_view($node, $view_mode, $langcode) {
...
+    foreach ($tree as $item => $data) {
+      $topic_names[] =  l($data->name, "/taxonomy/term/" . $data->tid);
+    }
+
+    $variables = array(
+      'root' => $topic,
+      'tree' => implode(', ', $topic_names),
+    );
+    $node->content['c4m_vocab_topic'][$index]['#markup'] = theme('c4m_og_vocab', $variables);
+
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/content/c4m_content_group/c4m_content_group.module]: move this empty line 1 line above ;)

**Comment by nirgn975** [comment]
@ordavidil Ready for review.

File: project/profiles/capacity4more/modules/c4m/content/c4m_content_group/c4m_content_group.module [hunk 6]
```diff
@@ -842,4 +855,23 @@ function c4m_content_group_node_view($node, $view_mode, $langcode) {
...
   );
+
+  // Style the Topics vocab.
+  foreach($node->content['c4m_vocab_topic']['#items'] as $index => $item) {
+    $topic = $item['taxonomy_term'];
+
+    // Create the topics tree.
+    $topic_names = array();
+    $tree = hierarchical_taxonomy_get_tree($topic->vid, $topic->tid);
+    foreach ($tree as $node => $term) {
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/content/c4m_content_group/c4m_content_group.module]:
```php
foreach ($tree as $term) {
```

**Comment by nirgn975** [comment]
@ordavidil Ready for review.

**Comment by ordavidil** [comment]
@nirgn975 
Please re-roll

**Pull request closed without merging**